src/glsl/material/fragments/flat.glsl

```diff
@@ -7,20 +7,25 @@
 // Material implementation header guard.
 //
 #ifdef useFlatMaterial
-
 //
 // Shader entry.
 //
 #pragma glslify: export(main)
+
 void main() {
   GeometryContext geometry = getGeometryContext();
   gl_FragColor = material.color;
+
 #ifdef HAS_MAP
   if (map.resolution.x > 0.0 && map.resolution.y > 0.0) {
     gl_FragColor = texture2D(map.data, geometry.uv);
   }
 #endif
 
+#ifdef HAS_ENV_MAP
```

Comment:
@vipyne we'll have to wait for the cube texture implementation first and then have another follow up PR to add this